src/s2geography/geoarrow_test.cc

```diff
@@ -1,9 +1,15 @@
-
+#include <gmock/gmock-matchers.h>
 #include <gtest/gtest.h>
 
 #include "nanoarrow/nanoarrow.hpp"
+#include "vendored/geoarrow/geoarrow.h"
```

Comment:
The "vendored" part is a hack that made it work locally because I still had this lying around in my conda from when s2geography installed those vendored headers (i.e. before https://github.com/paleolimbot/s2geography/pull/25)